guildwars2/account.py

```diff
@@ -318,19 +318,45 @@ def is_completed(encounter):
         longest = len(max(seq, key=len))
         if longest < 8:
             longest = 8
-        output = [
-            "LOCATION{}COUNT".format(" " * (longest - 5)),
-            "--------{}|-----".format("-" * (longest - 6))
-        ]
+        if 'is_upgrade' in choice and choice['is_upgrade']:
+            output = [
+                "LOCATION{}INV / GEAR".format(" " * (longest - 5)),
+                "--------{}|-----".format("-" * (longest - 6))
+            ]
+            align = 110
+        else:
+            output = [
+                "LOCATION{}COUNT".format(" " * (longest - 5)),
+                "--------{}|-----".format("-" * (longest - 6))
+            ]
+            align = 80
         total = 0
         storage_counts = OrderedDict(
             sorted(search_results.items(), key=lambda kv: kv[1], reverse=True))
+        characters = await self.get_all_characters(user)
+        char_names = []
+        for character in characters:
+            char_names.append(character.name)
         for k, v in storage_counts.items():
             if v:
-                total += v
-                output.append("{} {} | {}".format(k.upper(),
+                if 'is_upgrade' in choice and choice['is_upgrade']:
+                    total += v
+                    if k in char_names:
+                        slotted_upg = await self.collect_upgrades(ctx, k.title(), choice["ids"][0])
```

Comment:
That's going to be a lot of unnecesary API calls. Maybe we could add a new parameter to find_items_in_account that returns detailed results with per character equipped/not equipped?